content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -218,6 +289,27 @@ this variable to checkout the appropriate version of the library.
 
 image::pipeline/global-pipeline-library-legacy-scm.png["Configuring a 'Legacy SCM' for a Pipeline Library", role=center]
 
+==== Dynamic retrieval
```

Comment:
But a reader will not understand “retrieval methods” until they get to this point in the document.